lib/widgets/inbox.dart

```diff
@@ -537,13 +537,16 @@ class _TopicItem extends StatelessWidget {
               child: Text(
                 style: TextStyle(
                   fontSize: 17,
+                  // ignore: unnecessary_null_comparison // null topic names soon to be enabled
+                  fontStyle: topic.displayName == null ? FontStyle.italic : null,
```

Comment:
msglist: Display realmEmptyTopicDisplayName where empty topic appears
    
    Signed-off-by: Zixuan James Li <zixuan@zulip.com>

This is another NFC commit, right? (And it doesn't actually do what the summary line says it does, because empty topics don't appear anywhere yet?)

Reply:
Yeah, these commits are structured in a way so that they are not user visible without the final "do not merge" series. This way we can start merging part of it while still have a way to test them.

It will do what the summary says, but only after we allow empty topics in our data structures.